.circleci/config.yml

```
@@ -156,6 +156,8 @@ workflows:
           context: smarthr-dockerhub
       - chromatic-deployment:
           filters:
+            tags:
+              only: /^$/
```

`^$` で本当に動いて良いのか不安なので提案ｗ

```suggestion
              only: /^v.*/
```

---

LTGM!